pkg/memlimit/memlimit.go

```diff
@@ -0,0 +1,41 @@
+package memlimit
+
+import (
+	"os"
+	"runtime/debug"
+	"strconv"
+
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.LoggerForModule()
+
+	// setMemoryLimit specifies the function to use when setting the (soft) memory limit.
+	//
+	// This exists for testing purposes.
+	setMemoryLimit = debug.SetMemoryLimit
+)
+
+// SetMemoryLimit sets a (soft) memory limit on the Go runtime.
+// See debug.SetMemoryLimit for more information.
+func SetMemoryLimit() {
+	if goLimit := os.Getenv(`GOMEMLIMIT`); goLimit != "" {
+		log.Infof("GOMEMLIMIT set to %s", goLimit)
+		return
+	}
+
+	if roxLimit := os.Getenv(`ROX_MEMLIMIT`); roxLimit != "" {
```

Comment:
Out of curiosity, why not quote with `"`

Reply:
personal preference for consts